server/cli/launch.py

```diff
@@ -115,6 +118,9 @@ def launch(
     if extension != ".h5ad":
         raise click.FileError(basename(data), hint="file type must be .h5ad")
 
+    if developer:
```

Comment:
I think that we may not need all of debug, verbose, and developer, but it would take more time than we have to get the release out and is low priority.